Framework/PythonInterface/plugins/algorithms/HB2AReduce.py

```diff
@@ -341,18 +345,41 @@ def get_vanadium(self, detector_mask, m1, colltrans, exp, indir):
             # m1 = 0 -> Ge 115, 1.54A
             # m1 = 9.45 -> Ge 113, 2.41A
             # colltrans is the collimator position, whether in or out of the beam
-            # colltrans = 0 -> IN
-            # colltrans = +/-80 -> OUT
-            vcorr_filename = 'HB2A_{}__Ge_{}_{}_vcorr.txt'.format(
-                exp, 115 if np.isclose(m1, 0, atol=0.1) else 113,
-                "IN" if np.isclose(colltrans, 0, atol=0.1) else "OUT")
+            if self.check_date():
```

Comment:
`check_date` would be better if it was `get_date(filename)` and returned a python [`date` object](https://docs.python.org/3/library/datetime.html#date-objects). Then this if/else would compare to a `date.fromisoformat('2021-02-23')` and aid the future developer when they need to do something else based on an epoch.